### DISCUSSION TITLE
Lista de competências de cada membro evolvido no projeto.

### DISCUSSION THREAD

**Discussion by tallysdev** [tallysdev/eventSync#28] [General]
# Substitua sua competência anterior com uma nova nos comentários da issue
Antes: 

    Membro     |     Competências   |
    ---------  | ----------- |
    Italo Maurício      | Desenvolvedor Java
    Lucas Mateus        | Desenvolvedor Python
    Vinicius Maia       | Desenvolvedor TypeScript
    Tallys Aureliano | git, javascript, typescript, node, flutter, vue, canva, gerencia de requisitos, gerencia de atividades, mediação de conflitos |
    Dayanne Xavier      | Desenvolvedor C

Exemplo de resposta: 
Tallys Aureliano | git, javascript, typescript, node, flutter, vue, canva, gerencia de requisitos, gerencia de atividades, mediação de conflitos |
------- | -------- |

**Comment by italo-mauricio** [comment]
Italo Maurício  |  java, git, postgresql, comunicação, resolução de problemas.

**Comment by DayXL** [comment]
Dayanne Xavier | javascript, flutter, canva, C, python |

**Comment by mts-lucas** [comment]
Lucas Mateus | git, python, django, django rest framework, flutter, postgresql, mediação de conflitos, gerencia de pessoas |
------- | -------- |

**Comment by ViniciusMaiaM** [comment]
Vinicius Maia | git, typescript, node, python, flutter, postgresql, mediação de conflitos, boa comunicação |
------- | -------- |
